src/components/EnvelopePrimaryActions.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+	<ul class="list-item-content__actions--primary">
+		<!-- @slot Provide the primary actions for the right side quick menu, inside of Actions -->
+		<slot
+			name="primaryActions" />
```

Comment:
I'd suggest to replace the named slot by a default slot to simplify how the component is used

Reply:
@JuliaKirschenheuter :ping_pong: :)